ansible/roles/php/tasks/php-cli.yml

```diff
@@ -1,10 +1,10 @@
 ---
+- name: enabling opcache cli
+  lineinfile: dest=/etc/php/7.0/cli/php.ini
+              regexp=';?opcache.enable=\d'
+              line='opcache.enable=1'
```

Comment:
I added opcache out of habit :)
It's not necessary so I can remove this, but I don't see any disadvantages of enabling opcache